main/bash/bash.post-install

```diff
@@ -1,3 +1,3 @@
-#!/bin/sh
+#!/bin/bash
 add-shell '/bin/bash'
 exit 0
```

Comment:
why? I cannot see any good reason to why use bash for this?

Please revert this back to `#!/bin/sh`